client/my-sites/customer-home/cards/tasks/site-setup-list/get-task.js

```diff
@@ -168,20 +168,22 @@ export const getTask = (
 			taskData = {
 				timing: 10,
 				title: translate( 'Create a site menu' ),
-				description: translate(
-					"Building an effective navigation menu makes it easier for someone to find what they're looking for and improve search engine rankings."
+				description: (
+					<>
+						{ translate(
+							"Building an effective navigation menu makes it easier for someone to find what they're looking for and improve search engine rankings."
+						) }{ ' ' }
+						<InlineSupportLink
+							supportPostId={ 59580 }
+							supportLink={ localizeUrl( 'https://wordpress.com/support/menus/' ) }
+							showIcon={ false }
+							text={ translate( 'View tutorial.' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'View tutorial' )` *ES Score: 11*